src/main/java/com/commercetools/sync/inventory/SkuKeyTuple.java

```diff
@@ -0,0 +1,86 @@
+package com.commercetools.sync.inventory;
+
+import io.sphere.sdk.inventory.InventoryEntry;
+import io.sphere.sdk.inventory.InventoryEntryDraft;
+
+import javax.annotation.Nonnull;
+
+/**
+ * Helper class that holds sku and supply channel key - values used for {@link io.sphere.sdk.inventory.InventoryEntry}
```

Comment:
Not necessarily. It's more like part of `InventorySync` implementation so I would rather keep it package-private. There are assumptions that go well with my implementation but makes this class useless as public helper (e.g. taking supply channel key from supply channel's reference id when that reference isn't expanded)